src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -328,6 +328,14 @@ def to_model(
         )
         metadata = None
         if include_metadata:
+            is_templatable = False
```

Comment:
In the `get_custom_filters`, it seems like we are looking for pipeline builds that have `is_local` set to False. Should we include a similar logic here?

Reply:
Yep, added it here and also in one other place where we check something similar 👍